notebooks/community/model_evaluation/automl_tabular_regression_model evaluation.ipynb

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
<p>Move this cell one above. Add text why using BQ.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>